src/models/Movie/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as getAll } from './getAll';
+export { default as parseMovieData } from './parseMovieData';
```

Comment:
We don't need this barrel file if you don't want it. There are actually some case studies showing this pattern isn't great. Up to you ultimately, it doesn't matter for this exercise.

Reply:
https://marvinh.dev/blog/speeding-up-javascript-ecosystem-part-7/